node/src/actors/chain_manager/handlers.rs

```diff
@@ -886,7 +926,7 @@ impl Handler<PeersBeacons> for ChainManager {
         } else if pb_len < peers_needed_for_consensus {
             // Not enough outbound peers, do not unregister any peers
             log::debug!(
-                "Got {} peers but need at least {} to calculate the consensus",
+                "Got {} peers after filtering but need at least {} to calculate the consensus",
```

Comment:
After filtering by what criteria?

Reply:
Finally I remove the filter, so this comment modification is wrong